com.unity.render-pipelines.universal/Editor/ScriptableRendererDataEditor.cs

```diff
@@ -104,6 +104,18 @@ private bool GetCustumTitle(Type type, out string title)
             return false;
         }
 
+        private bool GetTooltip(Type type, out string tooltip)
+        {
+            var attribute = type.GetCustomAttribute<TooltipAttribute>();
+            if (attribute != null)
+            {
+                tooltip = attribute.tooltip;
+                return attribute != null;
```

Comment:
you can return directly a `true` and avoid the second null check, as it is expensive

Reply:
I don't see how that's expensive, but of course there's no need to have a duplicate check.
Fixed.